lib/user_info_scrubber.rb

```diff
@@ -20,7 +20,7 @@ def self.scrub_personal_info!(user)
   def self.scrub_details
     begin
```

Comment:
The retries here were to avoid a possibly duplicate on the display_name unique index. Add `@user.update_columns(email: nil)` to the scrub_personal_info! and remove all the `display_name` cruft now it's not needed.